XamMacSwiftUITest/AppDelegate.cs

```diff
@@ -14,7 +14,7 @@ public class AppDelegate : NSApplicationDelegate
 		public override bool ApplicationShouldTerminateAfterLastWindowClosed (NSApplication sender)
 		{
 			GC.Collect ();
-			return true;
+			return false;
```

Comment:
Why don't we want the app to terminate properly? Same in the F# sample?

Reply:
The app is crashing when you close the window, meaning we have another memory management issue. Having it terminate serves no purpose and hides these types of issues

Reply:
Oh another leak!